cookietemple/create/templates/cli/cli_python/{{ cookiecutter.project_slug_no_hyphen }}/.github/workflows/run_tests.yml

```diff
@@ -0,0 +1,201 @@
+name: Tests
+
+on:
+    - push
+    - pull_request
+{% raw %}
+jobs:
+    tests:
+        name: ${{ matrix.session }} ${{ matrix.python-version }} / ${{ matrix.os }}
+        runs-on: ${{ matrix.os }}
+        strategy:
+            fail-fast: false
+            matrix:
+                include:
+                    - {
+                          python-version: 3.9,
+                          os: ubuntu-latest,
+                          session: "pre-commit",
+                      }
+                    - {
+                          python-version: 3.9,
+                          os: ubuntu-latest,
+                          session: "safety",
+                      }
+                    - {
+                          python-version: 3.9,
+                          os: ubuntu-latest,
+                          session: "mypy",
+                      }
+                    - {
+                          python-version: 3.8,
+                          os: ubuntu-latest,
+                          session: "mypy",
+                      }
+                    - {
+                          python-version: 3.7,
+                          os: ubuntu-latest,
+                          session: "mypy",
+                      }
+                    - {
+                          python-version: 3.6,
```

Comment:
we may need to delete this here, as discussed earlier (drop python 3.6 support)

Reply:
3.10 will come out in April 

Ya, whatever we can drop 3.6